libs/SalesforceAnalytics/build.gradle.kts

```diff
@@ -77,6 +77,10 @@ android {
         buildConfig = true
     }
 
+    kotlin {
+        jvmToolchain(17)
```

Comment:
This is the first Kotlin in this module, so this matches our other Kotlin-enabled modules.